src/components/ColumnTitle/ColumnFilter.tsx

```diff
@@ -113,9 +121,14 @@ export const ColumnFilter: React.FC<ColumnFilterProps> = props => {
  */
 export function mapStateToProps(store: Store, ownProps: ColumnFilterOwnProps) {
     const widget = store.view.widgets.find(item => item.name === ownProps.widgetName)
+    const assocWidget = store.view.widgets.find(
+        item =>
+            item.bcName === ((ownProps.widgetMeta as unknown) as PickListFieldMeta).popupBcName && item.type === WidgetTypes.AssocListPopup
```

Comment:
`as unknown` conversion is not needed here as `ownProps.widgetMeta` is of correct union type. The reason why this conversion was required in `handleVisibleChange` is that there variable was already explicitly converted to something else (`MultivalueFieldMeta`)